includes/Core/Util/Authentication_Trait.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Authentication_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
```

Comment:
This trait should only ever be used for tests so this should be in `tests/phpunit/includes` instead.

E.g. https://github.com/google/site-kit-wp/blob/3167c6cbbf003ce9c52a8713b63f26b512f96b28/tests/phpunit/includes/OptionsTestTrait.php#L11

Reply:
Fixed